src/app/components/Footer/index.jsx

```diff
@@ -59,64 +61,28 @@ const StyledParagraph = styled.p`
   margin: 0;
 `;
 
-const listItems = [
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '1',
-    text: 'Why you can trust the BBC',
-  },
-  {
-    href: 'https://www.bbc.com/terms',
-    id: '2',
-    text: 'Terms of Use',
-  },
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '3',
-    text: 'About the BBC',
-  },
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '4',
-    text: 'Privacy Policy',
-  },
-  {
-    href: 'https://www.bbc.com/terms',
-    id: '5',
-    text: 'Cookies',
-  },
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '6',
-    text: 'Accessibility Help',
-  },
-  {
-    href: 'https://www.bbc.com/news/help-41670342',
-    id: '7',
-    text: 'Contact the BBC',
-  },
-];
-
-const externalLink = {
-  href: 'https://www.bbc.co.uk/help/web/links/',
-  text: 'Read about our approach to external linking.',
-};
-
-const Footer = () => (
+const Footer = ({ list, text, link }) => (
```

Comment:
These props are very ambiguous. It would be helpful if they were more specific to what they are used in.